mmv1/third_party/terraform/resources/resource_container_node_pool.go.erb

```diff
@@ -354,6 +354,21 @@ var schemaNodePool = map[string]*schema.Schema{
 					ValidateFunc: validateIpCidrRange,
 					Description:  `The IP address range for pod IPs in this node pool. Only applicable if create_pod_range is true. Set to blank to have a range chosen with the default size. Set to /netmask (e.g. /14) to have a range chosen with a specific netmask. Set to a CIDR notation (e.g. 10.96.0.0/14) to pick a specific range to use.`,
 				},
+				"network_performance_config": {
+					Type:        schema.TypeList,
+					Optional:    true,
+					MaxItems:    1,
+					Description: `Network bandwidth tier configuration.`,
+					Elem:        &schema.Resource{
+						Schema: map[string]*schema.Schema{
+							"total_egress_bandwidth_tier": {
```

Comment:
Would it make sense to add support for `externalIpEgressBandwidthTier` at the same time? (non-blocking)

Reply:
I see it's much less documented than total_egress_bandwidth_tier, I also played with it a little and got failures from GKE API, so I would prefer to stay with just this one.